game/lib/game_menu.py

```diff
@@ -0,0 +1,66 @@
+import pygame
+import pygame.freetype
+import json
+import os
+from lib import colors
+from lib import gamestates
+from lib import button
+
+game_config = None
+file_exists = os.path.isfile("config/game_config.json")
+if file_exists:
+    with open("config/game_config.json") as json_file:
+        game_config = json.load(json_file)
+
+FONT_STYLE = game_config['font'] if game_config is not None else "Segoe UI"
+FONT_SIZE_TITLE = game_config['menu_title_font_size'] if game_config is not None else 70
+FONT_SIZE_MENU = game_config['menu_font_size'] if game_config is not None else 50
+
+
+class GameMenu:
+    def __init__(self, game):
+        self.__game = game
+        self.__screen_size = self.__game.get_screen().get_size()
+        self.__text_menu_height = FONT_SIZE_MENU
+        self.__text_title_height = FONT_SIZE_TITLE
+        self.__text_menu_font = pygame.freetype.SysFont(FONT_STYLE, self.__text_menu_height)
+        self.__font_game_title = pygame.freetype.SysFont(FONT_STYLE, self.__text_title_height)
+        self.__empty_rect = 2
+        self.__title_pos_y = self.__screen_size[1] / 9
```

Comment:
indexy 1 i 0 wrzucić do zmiennych i nazwać. Tutaj chyba chodziło Ci o wyciągnięcie wartości X, Y. To nazwać 0, 1 jako X_INDEX, Y_INDEX. To samo niżej